docs/content/Configuration/Downstream/Jupyter.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Connecting to Jupyter
+permalink: /config/downstream/jupyter
+---
+
+You can connect to Cube from Jupyter using the [Cube SQL API][ref-sql-api]. The Jupyter Notebook is the original web application for creating and sharing computational documents.
```

Comment:
I know this is from the jupyter.org website, but maybe we remove the word "original"? It makes it sounds like other notebooks are "fake" 😄